GOVERNANCE.md

```
@@ -0,0 +1,258 @@
+**Examples of recognized contributions**
+
+There is no minimum amount of contributions needed to reach this level. You are just expected to show **sustained** involvement over a longer period of time (a couple months at least).
+
+* Championing a new feature in one of our libraries or championing a new library altogether using the RFC system
+* Helping to maintain a package
+* Submitting multiple merged PRs
+* Actively helping the community by enforcing the CoC or helping in support
+* Active participation in community events
+* Similar contributions
+
+**Privileges**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)